packages/fast-components-react-base/src/breadcrumb/breadcrumb.props.ts

```diff
@@ -0,0 +1,22 @@
+import * as React from "react";
+import {
+    BreadcrumbClassNameContract,
+    ManagedClasses,
+} from "@microsoft/fast-components-class-name-contracts-base";
+
+export interface BreadcrumbManagedClasses
+    extends ManagedClasses<BreadcrumbClassNameContract> {}
+export interface BreadcrumbUnhandledProps extends React.HTMLAttributes<HTMLDivElement> {}
+export interface BreadcrumbHandledProps extends BreadcrumbManagedClasses {
+    /**
+     * The breadcrumb children
+     */
+    children?: React.ReactNode;
+
+    /**
+     * The breadcrumb seprator
+     */
+    seperator?: JSX.Element;

```

Comment:
I think we wanted to use a render prop here, not just allow a JSX.Element? At worst, we should support React.ReactNode